### PR TITLE
Tighten up the language of the algorithms

### LINKED ISSUE #194: Byte and byte sequence vs octet/octets
Hi @annevk and @domenic 

The current spec uses octets a lot (as written by @zolkis) but I notice that INFRA uses byte and byte sequences. Any recommendation here?

### PR DESCRIPTION
**Pull request by kenchris**
- Tighten up the language and align them with modern standards
- Use byte instead of octet

Fixes #194

### CODE REVIEW

**Review by zolkis — APPROVED** [review]
Thanks!

File: index.html [hunk 1]
```diff
@@ -1854,18 +1856,16 @@ <h3>The <dfn>NFCReaderOptions</dfn> dictionary</h3>
           </li>
           <li>
             If there is no underlying <a>NFC Adapter</a>, or if a connection cannot
-            be established, then reject <var>p</var> with a <code>"<a>NotReadableError</a>"</code>
-            <code><a>DOMException</a></code> and return <var>p</var>.
+            be established, then reject <var>p</var> with a <code><a>NotReadableError</a></code>
+            and return <var>p</var>.
```
Comment (annevk) [inline index.html:1860]: This seems like a regression to me, FWIW.

Comment (kenchris) [reply index.html:1860]: This fits with now it is done in WHATWG URL spec

Comment (kenchris) [reply index.html:1860]: image

Or you want to make clear that it is a DOMException when used with a promise and not just thrown?

Comment (annevk) [reply index.html:1860]: A `TypeError` is not a `DOMException`, it's a separate class. Look at https://fetch.spec.whatwg.org/ for a specification that throws both `TypeError` and "`AbortError`" `DOMException`.

**Pull request merged**